source/geod24/concurrency.d

```diff
@@ -1031,9 +1034,14 @@ protected:
     {
         ThreadInfo info;
 
-        this(void delegate() op, size_t sz = 16 * 1024 * 1024) nothrow
+        public this (void delegate () op) nothrow
+        {
+            super(op);
+        }
+
+        public this (void delegate () op, size_t sz) nothrow
```

Comment:
I don't see why this change is necessary ? The old version covered both cases.

Reply:
I wanted to make it possible to set different stack sizes for each fiber.
But this doesn't matter, so I'll put it back the old way.